rules_building_block/discovery_capnetraw_capability.toml

```diff
@@ -0,0 +1,82 @@
+[metadata]
+bypass_bbr_timing = true
+creation_date = "2024/01/10"
+integration = ["endpoint"]
+maturity = "production"
+min_stack_comments = "Linux effective and permitted process capability data sources were added in version 8.11.0"
+min_stack_version = "8.11.0"
+updated_date = "2024/01/10"
+
+[rule]
+author = ["Elastic"]
+building_block_type = "default"
+description = """
+
+Identifies the ability of a process to be able to create RAW and PACKET socket types for the available network
+namespaces by a non-root user. A malicious process with this capability may exploit routing between hosts,
```

Comment:
May want to include something here about the logic behind using a new_terms rule here and the fields you chose to trigger on